code/game/gamemodes/changeling/powers/tiny_prick.dm

```diff
@@ -61,7 +61,7 @@
 	return 1
 
 
-/obj/effect/proc_holder/changeling/sting/transformation
+obj/effect/proc_holder/changeling/sting/transformation
```

Comment:
I noticed that but it seemed like all the other strings didn't have that first / but the transformation sting. Any reason why its different?